src/App.js

```diff
@@ -5,19 +5,19 @@ import firebase from 'firebase/app';
 import 'firebase/firestore';
 import 'firebase/auth';
 import 'firebase/analytics';
+import PropTypes from 'prop-types';
```

Comment:
This shouldn't be here this branch is specifically for adding eslint functionality, kindly create another branch to add this feature.